lib/HomeScreen.dart

```diff
@@ -0,0 +1,181 @@
+import 'dart:math';
```

Comment:
pr should contain only changes for additional tasks, not all files in the project